supervisor/resolution/const.py

```diff
@@ -37,6 +37,7 @@ class UnsupportedReason(str, Enum):
     PRIVILEGED = "privileged"
     SYSTEMD = "systemd"
     JOB_CONDITIONS = "job_conditions"
+    TRUST = "trust"
```

Comment:
We should use `CONTENT_TRUST = "content_trust"` both here and for `UnhealthyReason`

Reply:
I like unhealthyReason like it is because it matches better with the other const in the group and it's different.